src/components/v5/common/Fields/RadioButtons/ButtonRadioButtons/ButtonRadioButtons.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import Icon from '~shared/Icon';
+import RadioButtonsBase from '../RadioButtonsBase';
+import { ButtonRadioButtonsProps } from './types';
+import { RadioItem } from '../RadioButtonsBase/types';
+
+const ButtonRadioButtons: FC<ButtonRadioButtonsProps> = ({
+  items,
+  className,
+  ...rest
+}) => {
+  const modifiedItems = items.map<RadioItem<string>>(
+    ({ label, iconName, colorClassName, ...item }) => ({
+      ...item,
+      // eslint-disable-next-line react/no-unstable-nested-components
```

Comment:
is it necessary?

Reply:
Yes, because TS thinks that it's a component when it's not